aws-py-ec2-provisioners/provisioners.py

```diff
@@ -65,6 +65,9 @@ def connect(conn: ConnectionArgs) -> paramiko.SSHClient:
 class ProvisionerProvider(dynamic.ResourceProvider):
     __metaclass__ = abc.ABCMeta
 
+    def ignore_properties(self) -> List[str]:
```

Comment:
Is there a more idiomatic fix? The integration test was verifying that the empty preview remains empty, it wasn't true because the `results` property appeared to be being removed for the diff on `cat_config = provisioners.RemoteExec('cat-config',...)`

Reply:
How does the TS version compare?